java/com/google/javascript/jscomp/Diagnostics.java

```diff
@@ -87,12 +89,15 @@
           "duplicate",
           "es5Strict",
           "externsValidation",
+          "extraProvide",
           "extraRequire",
           "fileoverviewTags",
           "globalThis",
+          "googModuleExportNotAStatement",
           "invalidCasts",
-          "lateProvide",
           "legacyGoogScopeRequire",
+          "lateProvide",
+          "lintChecks",
```

Comment:
Looking at history, I think this was excluded on purpose; I don't want to dig why. Let's keep it for now.

Reply:
Again I'm not sure if you're telling me to remove this line or not.